src/Equations/RunawayFluid.cpp

```diff
@@ -866,6 +866,28 @@ real_t* RunawayFluid::evaluatePartialContributionAvalancheGrowthRate(len_t deriv
     return dGamma;
 }
 
+/**
+ * Same as the function above, but for the formula published by Hesslow et al NF 2019
+ */
+real_t* RunawayFluid::evaluatePartialContributionAvalancheGrowthRateAlt(len_t derivId) {
+    real_t *dGamma = new real_t[nr];
+    if( !( (derivId==id_Eterm) || (derivId==id_ntot) ) )
+        for(len_t ir = 0; ir<nr; ir++)
+            dGamma[ir] = 0;
```

Comment:
What is the purpose of this code? It seems it never has any effect, since ``dGamma`` is always overwritten on line 880.

Reply:
Looks like a logic error (that I introduced). Should be followed by return dGamma after the for loop (inside the if statement)